codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -21,13 +21,24 @@ def validate(sel, input: TermsAgreementInput):
             raise ValidationError("Terms of agreement cannot be null")
 
     def update_terms_agreement(self, input: TermsAgreementInput):
+        if not self.current_owner or not self.current_owner.user_id:
+            raise Unauthenticated()
+
+        ts = timezone.now()
```

Comment:
Nitpick: we try to keep variable names representative of what they are, how would you feel about calling this `now`?

Reply:
Done.